torchvision/ops/_custom_ops.py

```diff
@@ -0,0 +1,44 @@
+import os
+import torch
+import glob
+
+# load the custom_op_library and register the custom ops
+custom_op_lib_path = os.path.join(os.path.dirname(__file__), '..', 'custom_ops.*.so')
+custom_op_lib = glob.glob(custom_op_lib_path)
+torch.ops.load_library(custom_op_lib[0])
+
+
+def register_custom_op():
+    from torch.onnx.symbolic_helper import parse_args, scalar_type_to_onnx
+    from torch.onnx.symbolic_opset9 import select, unsqueeze, squeeze, _cast_Long, reshape
+
+    @parse_args('v', 'v', 'f')
+    def symbolic_multi_label_nms(g, boxes, scores, iou_threshold):
+        boxes = unsqueeze(g, boxes, 0)
+        scores = unsqueeze(g, unsqueeze(g, scores, 0), 0)
+        max_output_per_class = g.op('Constant', value_t=torch.tensor([2000], dtype=torch.long))
```

Comment:
This seems like something that should be customizable, or the largest possible number? As is, it looks like this could lead to subtle bugs down the line